web-app/src/components/Game.js

```diff
@@ -790,6 +747,58 @@ export default class Game extends React.Component {
 											),
 									  ]
 									: ""}
+								<Divider />
+								<MenuItem
+									key="game-id"
+									onClick={(_) => {
+										const hrefURL = new URL(location.href);
+										helpers
+											.copyToClipboard(
+												hrefURL.protocol +
+													"//" +
+													hrefURL.host +
+													"/Game/" +
+													this.state.game.Properties
+														.ID
+											)
+											.then(
+												(_) => {
+													this.setState({
+														moreMenuAnchorEl: null,
+													});
+													
+													if (this.state.game.Properties.Started) {
+														helpers.snackbar(
+															"Game URL copied to clipboard. Share it to show the game."
+														)} else {
+														{
+														helpers.snackbar(
+															"Game URL copied to clipboard. Share it to invite other players."
+														)}
+													};
+												},
+												(err) => {
+													console.log(err);
+												}
+											);
+										gtag("event", "game_share");
+									}}
+								>
+									{this.state.game.Properties.Started ? ("Share game") : ("Invite players")}
```

Comment:
"Invite players" will make the player believe it actually sends invitations. Sharing is closer to the truth when you just get a link to the game, don't you think?